workspaces/ui-v2/src/components/navigation/NavButton.tsx

```diff
@@ -13,9 +14,15 @@ export type NavButtonProps = {
 export function NavButton(props: NavButtonProps) {
   const classes = useStyles();
   const { Icon, title, to } = props;
+  const history = useHistory();
 
   return (
-    <Link className={classes.root} href={to}>
+    <Link
```

Comment:
@JaapRood I know you changed this in your previous PR - realized as part of building this, we need to actually use onClick so we use the react-router redirect rather than the full page browser redirect.

I _think_ the button / link should maintain the appropriate accessibility / keyboard utility (I think material UI should handle this correctly)